### ISSUE TITLE
Default to latest version if no version is specified

### ISSUE THREAD

**Issue by shafu0x** [mario-eth/soldeer#103]
You have to define the version you want when installing a new package with `soldeer install`.
Maybe it could just default to the latest version if no version is specified.

So:
`soldeer install solmate~6.7.0`

becomes

`soldeer install solmate` (which defaults to 6.7.0)

**Comment by Mario-SO** [comment]
I would agree on defaulting to latest version. Already voted on the twitter poll.

Way more dev friendly imho. If you really need a specific version of some library, your mind already gets into "how to install this specific version", and you will easily find out the command. Very opinionated argument though, so very loosely held 😅

**Comment by EmperorOrokuSaki** [comment]
taking this on!

**Comment by beeb** [comment]
Hey @EmperorOrokuSaki , just checking in if you're still interested in working on this?

**Comment by beeb** [comment]
In the end we decided against this default behavior, because it encourages typosquatting and is generally less secure than going to soldeer.xyz to check what the last version is manually (it allows to inspect the dependency metadata, etc).
